testsuite/tests/lib-bigarray/bigarrays.ml

```diff
@@ -150,6 +150,14 @@ let _ =
                   Complex.i, Complex.i;
                   {im=0.5;re= -2.0}, {im=0.5;re= -2.0};
                   {im=3.1415;re=1.2345678}, {im=3.1415;re=1.2345678}]);
+  test 20 true
+    (test_setget int_least31
+                 [0, 0;
+                  123, 123;
+                  -456, -456;
+                  max_int, max_int land 0xFFFFFFFF;
```

Comment:
Note that the constant `0xFFFF_FFFF` will not compile on 32-bit. One solution is to do `max_int land (Int32.to_int 0xFFFF_FFFFl)`